certbot-dns-google/setup.py

```diff
@@ -0,0 +1,70 @@
+import sys
+
+from setuptools import setup
+from setuptools import find_packages
+
+
+version = '0.15.0.dev0'
+
+# Please update tox.ini when modifying dependency version requirements
+install_requires = [
+    'acme=={0}'.format(version),
+    'certbot=={0}'.format(version),
+    'fasteners',
```

Comment:
I think this is here as a result of a bad merge as I don't see why this package depends on `fasteners`.